src/sentry/api/endpoints/org_auth_tokens.py

```diff
@@ -43,7 +42,7 @@ def get(self, request: Request, organization: Organization) -> Response:
 
     def post(self, request: Request, organization: Organization) -> Response:
         jwt_token = generate_token(organization.slug, generate_region_url())
-        token_hashed = hashlib.sha256_text(jwt_token).hexdigest()
+        token_hashed = hash_token(jwt_token)
```

Comment:
Let's rename from `jwt_token` to `token` considering it's no longer a JWT token by format?